client/signup/steps/plans/index.jsx

```diff
@@ -349,8 +349,11 @@ export default connect(
 		showTreatmentPlansReorderTest:
 			'treatment' === plans_reorder_abtest_variation || isTreatmentPlansReorderTest( state ),
 		isLoadingExperiment: isLoading( state ),
+		// IMPORTANT NOTE: The following is always set to true. It's a hack to resolve the bug reported
+		// in https://github.com/Automattic/wp-calypso/issues/50896, till a proper cleanup and deploy of
+		// treatment for the `vertical_plan_listing_v2` experiment is implemented.
 		isInVerticalScrollingPlansExperiment:
-			'treatment' === getVariationForUser( state, 'vertical_plan_listing_v2' ),
+			true || 'treatment' === getVariationForUser( state, 'vertical_plan_listing_v2' ),
```

Comment:
I think we can just set this to true and remove rest of the code.

Reply:
Yup, I left it there intentionally as an indication to anybody looking at the code that it's a hack, and as a reminder for a cleanup. The experiment slug present there will help them look it up in Abacus and reach the experiments P2 post. I guess this line can be removed when the proper cleanup is worked on, what do you think?

Reply:
I went ahead with the change you suggested since I realized that the code comment calls out that it's a temporary fix and mentions the experiment slug.